lib/widgets/component/quick_menu.dart

```diff
@@ -37,54 +38,54 @@ class DrawerComponentState extends State<DrawerComponent> {
             title: Text('Hall Building'),
             onTap: () {
               Navigator.pop(context);
-              // accesses the sink of the LocationBloc provided to HomeScreen
-              BlocProvider.of<LocationBloc>(context)
-                  .locationCoordinates
-                  .add(LatLng(concordia_constants.H_BUILDING_LATITUDE, concordia_constants.H_BUILDING_LONGITUDE));
+              bloc.add(CameraMove(
+                  LatLng(concordia_constants.H_BUILDING_LATITUDE, concordia_constants.H_BUILDING_LONGITUDE),
```

Comment:
17.5 shows up 3+ times. Needs to be in a constant

Reply:
Fixed!